Web UI/CognitiveSearch.UI/Search/DocumentSearchClient.cs

```diff
@@ -66,7 +66,7 @@ public DocumentSearchResult<Document> Search(string searchText, SearchFacet[] se
         {
             try
             {
-                SearchParameters sp = GenerateSearchParameters(searchFacets, selectFilter, currentPage, polygonString);
+                SearchParameters sp = GenerateSearchParameters(searchFacets, selectFilter, currentPage, polygonString, false);
```

Comment:
Just curious, what does this last parameter do that we are setting to false, consider adding some documentation for method signature.

Reply:
Got the answer in next cell, please disregard this message